src/test/java/sh/blake/niouring/IoUringFileTest.java

```diff
@@ -32,4 +33,37 @@ public void open_and_read_readme_should_succeed() {
 
         Assert.assertTrue("File read successfully", readSuccessfully.get());
     }
+
+    @Test
+    public void seek_read_readme_should_succeed() {
+        // This test is fragile and will need to be maintained
+        // as the README is updated.
+        String fileName = "README.md";
```

Comment:
Instead of relying on the README here, can we add a `src/test/resources/file-test.txt` to make this test a little more robust? Let's also include a warning in that file to let people know that changing the contents could break some unit tests.